app/src/main/java/com/space/movieapp/presentation/home/ui/HomeFragment.kt

```diff
@@ -89,9 +91,17 @@ class HomeFragment : BaseFragment<HomeViewModel>() {
         }
     }
 
-    private fun navigationToFav() {
+    private fun navigationToDetails() {
         binding.movieTextview.setOnClickListener {
-            viewModel.navigationToFav()
+            viewModel.navigationToDetails()
+        }
+    }
+
+    private fun setFavoriteListener() {
+        moviesPagingAdapter.setOnIconClickListener {
+            lifecycleScope {
+                viewModel.toggleFavoriteMovie(it)
+            }
         }
```

Comment:
ფრაგმენტში რატო გაქ სკოუპში ეს viewModel.toggleFavoriteMovie(it)

Reply:
აუ ვიუმოდელში მქონდა სასფენდ ფუნქცია და ამიტო. შევცვალე. მადლობა!